### ISSUE TITLE
Cannot change tag of an existing source

### ISSUE THREAD

**Issue by Anthony25** [SSilence/selfoss#622]
Hello,
I can't change the tag of an existing source, otherwise I get this error
```
Internal Server Error

PDOStatement: ERREUR: id delimited by null length on or next to « "" » LINE 1: ...SET title=$1, tags=$2, spout=$3, params=$4, error="" WHERE i... ^
```

My system is setup in french, sorry if the translation of this error message is not exact. It's the same if I import an OPML.
I tested on a clean install, created a new source and tried to add or remove its tag and same error.

ArchLinux
Selfoss 2.12 (same thing on 2.13)
PostgreSQL 9.4.1
PHP 5.6.6
NGinx 1.6.2

Thanks !

**Comment by niol** [comment]
This can be closed: fixed in 2.13.

**Comment by Anthony25** [comment]
Yes, I confirm that it is fixed in 2.13.

Thanks !
